PROCESSORS.md

```diff
@@ -935,6 +936,35 @@ In the list below, the names of required properties appear in bold. Any other pr
 | gcs.error.domain   | failure      | The domain of the error occurred during operation.      |
 
 
+## FetchModbusTcp
+
+### Description
+
+Processor able to read data from industrial PLCs using Modbus TCP/IP
+
+### Properties
+
+In the list below, the names of required properties appear in bold. Any other properties (not in bold) are considered optional. The table also indicates any default values, and whether a property supports the NiFi Expression Language.
+
+| Name                           | Default Value | Allowable Values | Description                                                                                                                                                                                      |
+|--------------------------------|---------------|------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Hostname**                   |               |                  | The ip address or hostname of the destination.<br/>**Supports Expression Language: true**                                                                                                        |
+| **Port**                       | 502           |                  | The port or service on the destination.<br/>**Supports Expression Language: true**                                                                                                               |
+| **Unit Identifier**            | 0             |                  | The port or service on the destination.<br/>**Supports Expression Language: true**                                                                                                               |
+| **Idle Connection Expiration** | 15 seconds    |                  | The amount of time a connection should be held open without being used before closing the connection. A value of 0 seconds will disable this feature.<br/>**Supports Expression Language: true** |
```

Comment:
this looks like it was generated before `Supports Expression Language` got removed from `Idle Connection Expiration`

Reply:
👍 good catch, [update PROCESSORS.md](https://github.com/apache/nifi-minifi-cpp/pull/1779/commits/1cb3826d009971b5462c555014172ff36915bd56)